lib/msal-node/docs/initialize-confidential-client-application.md

```diff
@@ -16,15 +16,19 @@ In order to use MSAL Node, you need to instantiate a [ConfidentialClient](https:
 
 ```javascript
 import * as msal from "@azure/msal-node";
+// Due to security reasons, secrets should not be hardcoded.
```

Comment:
Same as above, add this before the example code as a tip/warning to help with usage.